learn/platform/networks/moonbeam.md

```diff
@@ -38,18 +38,18 @@ Some important variables/configurations to note include (still subject to change
     |     Cool-off Period      |      {{ networks.moonbeam.democracy.cool_period.blocks}} blocks ({{networks.moonbeam.democracy.cool_period.days}} days)      |
-    |     Minimum Deposit      |                                    {{ networks.moonbeam.democracy.    min_deposit }} MOVR                                    |
-    |      Maximum Votes       |                                       {{ networks.moonbeam.    democracy.max_votes }}                                        |
-    |    Maximum Proposals     |                                     {{ networks.moonbeam.democracy.    max_proposals }}                                      |
+    |     Minimum Deposit      |                                      {{ networks.moonbeam.democracy.min_deposit }} GLMR                                      |
+    |      Maximum Votes       |                                         {{ networks.moonbeam.democracy.max_votes }}                                          |
+    |    Maximum Proposals     |                                       {{ networks.moonbeam.democracy.max_proposals }}                                        |
 
 === "Staking"
-    |             Variable             |                                                    Value                                                    |
-    |:--------------------------------:|:-----------------------------------------------------------------------------------------------------------:|
-    |     Minimum nomination stake     |                          {{ networks.moonbeam.staking.    min_del_stake }} tokens                           |
-    | Maximum nominators per collators |                             {{ networks.moonbeam.staking.    max_del_per_can }}                             |
-    | Maximum collators per nominator  |                             {{ networks.moonbeam.staking.    max_del_per_del }}                             |
-    |              Round               | {{ networks.moonbeam.staking.round_blocks }} blocks ({{     networks.moonbeam.staking.round_hours }} hours) |
-    |          Bond duration           |                            {{ networks.moonbeam.staking.    bond_lock }} rounds                             |
+    |             Variable              |                                                  Value                                                  |
+    |:---------------------------------:|:-------------------------------------------------------------------------------------------------------:|
+    |     Minimum delegation stake      |                          {{ networks.moonbeam.staking.min_del_stake }} tokens                           |
+    | Maximum delegators per candidates |                             {{ networks.moonbeam.staking.max_del_per_can }}                             |
+    | Maximum candidates per delegator  |                             {{ networks.moonbeam.staking.max_del_per_del }}                             |
```

Comment:
Can we change this to maximum delegations per account? Would clarify the distinction between this and the above parameter.